molecule/multiserver/molecule.yml

```diff
@@ -0,0 +1,22 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+platforms:
+  - name: centos7
+    image: docker.io/pycontribs/centos:7
+    pre_build_image: true
+    vpn_ip: 10.16.0.1/32
```

Comment:
I get the following error when I run the `multiserver` test, so these variable assignments seem to not work:
```
fatal: [centos7]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'vpn_ip' is undefined"}
fatal: [centos8]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'vpn_ip' is undefined"}
fatal: [debstable]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'vpn_ip' is undefined"}
```

Reply:
Multiserver is WiP, maybe I can remove those files from the commit and PR, I haven't finished testing.